Readme.md

```diff
@@ -35,6 +35,14 @@
 </p>
 <br/>
 
+> **Note on Android support 📱**<br/>
+> The Slate community needs your help to add Android support to the latest version of Slate. We’ve successfully added Android support to a previous version of Slate, but due to an important but immense rewrite, the previous Android fixes could not be ported over.<br/><br/>
+> Due to the high time and monetory cost required to create the previous Android support we have resorted to trying to raise CAD \$30,000 to fund the development and testing of the new Android implementation. (We are currently at \$17,000!)<br/><br/>
```

Comment:
Maybe mention that Slates API has started to stabilise so it's a good time to add in Android Support. I don't know how true that is but I'm pretty sure TheSunny had reached out with Ian to double check. Otherwise it's looking pretty good 👍